.gitignore

```diff
@@ -90,3 +90,4 @@ feedstocks/*
 coverage.xml
 .coverage.*
 miniconda.sh
+README.md
```

Comment:
Why is this change?

Reply:
Running the `dashboard.py` script in this repo creates that file by default, so I added it to `.gitignore` so it won't overwrite the current `README.rst` and show up on this GitHub repo when it isn't supposed to.

Reply:
Ah, got it. Makes sense.